### ISSUE TITLE
Unable to read text of selected URL in address bar dropdown

### ISSUE THREAD

**Issue by glward** [mr-islam/zen-fox#15]
Not sure when this happened (recently, maybe with in the last week), but the background and foreground colours for the address bar current selection are the same, so it's not possible to read the selected URL:

![Screen Shot 2019-08-01 at 19 09 36](https://user-images.githubusercontent.com/494016/62333130-448ea480-b490-11e9-8c2c-369a55e54002.png)
![Screen Shot 2019-08-01 at 19 10 31](https://user-images.githubusercontent.com/494016/62333138-48222b80-b490-11e9-931b-2b4d4144b61c.png)

**Comment by mr-islam** [comment]
Oh my, I'm really sorry about this issue. Thanks for reporting it! 

Just within the past week, I updated to comply with the latest changes for Firefox 70 (since they're renaming some of the theming API variables) and also tried to theme new things like the text in the address bar dropdown. 

In my own testing this issue did not come up, but I just installed a fresh edition of Firefox and found the same error. I think my default Firefox configuration is a bit too customized to be reliable for testing 😅

Just pushed an update now, it should be good now. Thanks again 👍
